### ISSUE TITLE
Repository information

### ISSUE THREAD

**Issue by PyvesB** [eclipse/lsp4j#365]
This repository doesn't have any description, website nor topics:
![lsp4j](https://user-images.githubusercontent.com/10694593/64474621-3575d300-d16f-11e9-9d61-0e65a5901e4d.png)

Such information is useful when performing searches on GitHub, increases the visibility of the repository and gives users who aren't familiar with the project a quick summary of what it's about. Let's get this fixed!

**Comment by jonahgraham** [comment]
Good idea. We need to decide what we want and then ask @eclipsewebmaster to do the change because AFAICT only webmaster has access to edit.

How about:

- Description: Eclipse LSP4J is a Java implementation of VSCode's language server protocol intended to be consumed by tools and language servers implemented in Java.
- Website: https://eclipse.org/lsp4j
- Topics: lsp eclipse java

Any comments or objections before I raise this with webmaster?

**Comment by PyvesB** [comment]
Sounds good overall! A few minor observations:
* should we really mention VSCode? The language server protocol is a separate specification if I'm not mistaken, VSCode just happens to be one of the IDEs implementing it.
* I would suggest shortening the description slightly by removing "Eclipse LSP4J" at the start. The name of the repo will anyway appear just above, whether it be on the repo itself or in searches.
* the [language-server-protocol topic](https://github.com/topics/language-server-protocol) seems somewhat more common than its [lsp](https://github.com/topics/lsp) counterpart.

**Comment by KamasamaK** [comment]
I agree that "VSCode" does not need to be mentioned.

**Comment by jonahgraham** [comment]
Thanks for the quick feedback! I'll give this a couple of work days to review and then move it up the chain to webmaster.
